installation.md

```diff
@@ -152,6 +152,368 @@ After installation, go to the "Installed Apps" page and click "Open" to access t
+
+Before applying these manifests, make sure to:
+- Create or choose a namespace and replace `my-namespace` in the apply commands below
+- Replace `my-server-hostname` with your actual node hostname
+- Update `/mnt/pg-data`, `/mnt/ebk-storage`, and `/mnt/ebk-logs` paths to your desired storage locations
+- Generate secure values for secrets in `secret.yaml` (use `echo -n "secret-value" | base64`)
+- Update `ebk.my-domain.com` with your actual domain name
+- Ensure you have NGINX Ingress Controller installed in your cluster
+- Ensure you have cert-manager installed with a configured ClusterIssuer named `letsencrypt-cluster` for automatic TLS certificate provisioning
+- The manifests use `local-storage` StorageClass. Either create this StorageClass or update `storageClassName` in all manifests to use your cluster's storage solution (e.g., `longhorn`, `nfs-client`, `ceph-rbd`, or your cloud provider's default storage class)
+
+### secret.yaml
+
+Contains sensitive data including the PostgreSQL database password and ezBookkeeping's secret key for session encryption.
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ebk-secret
+type: Opaque
+data:
+  # echo -n "secret-value" | base64
+  POSTGRES_PASSWORD: XXX
+  EBK_SECURITY_SECRET_KEY: XXX
+```
+
+### pg.yaml
+
+PostgreSQL database deployment with StatefulSet, headless Service, and PersistentVolume for data storage.
+
+```yaml
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: "pg-data-pv"
+  labels:
+    pv-for: "pg-data"
+spec:
+  nodeAffinity:
+    required:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: kubernetes.io/hostname
+          operator: In
+          values:
+          - "my-server-hostname"
+  capacity:
+    storage: 10Gi
+  accessModes:
+    - ReadWriteOnce
+  persistentVolumeReclaimPolicy: Retain
+  storageClassName: local-storage
+  local:
+    path: "/mnt/pg-data"  # change to your actual path
+
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: "pg"
+spec:
+  serviceName: "pg"
+  replicas: 1
+  selector:
+    matchLabels:
+      app: "pg"
+  template:
+    metadata:
+      labels:
+        app: "pg"
+    spec:
+      nodeSelector:
+        kubernetes.io/hostname: "my-server-hostname"
+      containers:
+        - name: postgres
+          image: postgres:18
+          env:
+          - name: POSTGRES_USER
+            value: "ezbookkeeping"
+          - name: POSTGRES_DB
+            value: "ezbookkeeping"
+          - name: POSTGRES_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: "ebk-secret"
+                key: POSTGRES_PASSWORD
+          ports:
+          - containerPort: 5432
+          volumeMounts:
+          - mountPath: /var/lib/postgresql
+            name: "pg-data"
+          resources:
```

Comment:
I suggest removing all `resources` configuration, as everyone's requirements and hardware vary, and including them here could be misleading.